src/Polly.Core/Simmy/Latency/LatencyStrategyOptions.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Polly.Simmy.Latency;
+
+#pragma warning disable CS8618 // Required members are not initialized in constructor since this is a DTO, default value is null
+
+/// <summary>
+/// Represents the options for the Latency chaos strategy.
+/// </summary>
+public class LatencyStrategyOptions : MonkeyStrategyOptions
+{
+    /// <summary>
+    /// Gets or sets the delegate that's raised when delay occurs.
+    /// </summary>
+    /// <remarks>
+    /// Defaults to <see langword="null"/>.
+    /// </remarks>
+    public Func<OnDelayedArguments, ValueTask>? OnDelayed { get; set; }
+
+    /// <summary>
+    /// Gets or sets the latency generator that generates the delay for a given execution.
+    /// </summary>
+    /// <remarks>
+    /// Defaults to <see langword="null"/>. Either <see cref="Latency"/> or this property is required.
+    /// When this property is <see langword="null"/> the <see cref="Latency"/> is used.
+    /// </remarks>
+    public Func<ResilienceContext, ValueTask<TimeSpan>> LatencyGenerator { get; set; }
+
+    /// <summary>
+    /// Gets or sets the delay for a given execution.
+    /// </summary>
+    /// <remarks>
+    /// Defaults to <see langword="null"/>. Either <see cref="LatencyGenerator"/> or this property is required.
+    /// When this property is <see langword="null"/> the <see cref="LatencyGenerator"/> is used.
+    /// </remarks>
+    public TimeSpan? Latency { get; set; }
```

Comment:
Maybe we can use the same defaults as here:

https://github.com/dotnet/extensions/blob/0b95c32772a198137ca9be16ae61ec40ece984dc/src/Libraries/Microsoft.Extensions.Resilience/FaultInjection/Options/LatencyPolicyOptions.cs#L12

(these defaults are used across many production services at MS)